audio/src/fetch/mod.rs

```diff
@@ -300,6 +300,14 @@ impl StreamLoaderController {
         // terminate stream loading and don't load any more data for this file.
         self.send_stream_loader_command(StreamLoaderCommand::Close);
     }
+
+    pub fn from_local_file(file_size: u64) -> Result<Self, Error> {
```

Comment:
So why is this a Result?

Reply:
Good question... changed to return the type directly